recipes/cpr/patches/find_package_curl.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 0c10971..0e1ab2d 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -151,7 +151,7 @@ endif()
+ # Curl configuration
+ if(CPR_FORCE_USE_SYSTEM_CURL)
+     if(CPR_ENABLE_SSL)
+-        find_package(CURL COMPONENTS HTTP HTTPS)
++        find_package(CURL REQUIRED)
```

Comment:
This is perfectly fine, but you can also use the `-DCMAKE_REQUIRE_FIND_PACKAGE_CURL:BOOL=ON` CMake option (see https://cmake.org/cmake/help/latest/variable/CMAKE_REQUIRE_FIND_PACKAGE_PackageName.html#variable:CMAKE_REQUIRE_FIND_PACKAGE_%3CPackageName%3E). The main advantage of the variable w.r.t. to the patch is that you do not need to adapt the patch for future source code changes.

Reply:
Mhh, that is a different issue then. Do you have any idea why that is the case? Are the `HTTP` and `HTTPS` componentents missing from conda-forge curl build for any reason?

Reply:
👍 

Done